src/protocol/ipa_prf/prf_sharding/mod.rs

```diff
@@ -1062,4 +1066,68 @@ pub mod tests {
             assert_eq!(result, &expected);
         });
     }
+
+    #[test]
+    fn semi_honest_aggregation_capping_attribution_with_attribution_window() {
+        // In OPRF, the attribution window's upper bound is excluded - i.e., [0..ATTRIBUTION_WINDOW_SECONDS).
```

Comment:
Once @danielmasny's PR lands (https://github.com/private-attribution/ipa/pull/844) we will have access to both "greater than" and "greater than or equal to" comparisons, and it need not be this way.